### DISCUSSION TITLE
Sync assignments feature not working as expected

### DISCUSSION THREAD

**Discussion by john-french** [community/community#144436] [GitHub Education]
### Hi everyone,

Recently I needed to update starter code for an assignment which some students had already accepted via Github Classroom, so I tried to use Github Classroom's Sync Assignments feature.

I made the change to the starter code repo, then hit the "Sync assignments" button on the assignment in GH Classroom. I then went to a student repo and noticed that a new PR had been created. Happy days.

I expected that, when I merged this PR in the student's repo, the changes I had made to the starter code would be merged into the student's repo. But to my great surprise, that is not what happened.

Instead, the student's changes were merged to the starter code repo! Shortly after, another student accepted the assignment on Github classroom and received the starter code, with the other student's commit history included! 

Thankfully I was alerted to this very quickly and was unable to undo the mess. 

What's going on here? It's highly unintuitive, not to mention unhelpful, that a feature that (I thought?) was meant to pull changes from the starter code repo into student repos would do the exact opposite.

**Comment by darkrsw** [comment]
Same thing happens here still!
What is going on when I click "sync assignment"? Why the code of students are being merged in to my starter code?????

**Comment by dburchill** [comment]
I'm having the same problem. I make a change, push it, and then try to sync, I get this message "ErrorFailed to open 60 of 60 pull requests. This may mean that pull requests to sync your changes have already been successfully opened to your students repositories. If this is not the case, please try again or contact support."  Has anyone found a workaround for this?
